source/usage-examples/changeStream.txt

```diff
@@ -7,76 +7,80 @@ Watch for Changes
 Open a Change Stream
 --------------------
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-You can open a change stream by calling the ``watch()`` method on
-a ``Collection``, ``Db``, or ``MongoClient`` object. The change stream
-returns **change event** documents when they occur.
+You can watch for changes to a single collection, a database, or an entire deployment in MongoDB with
+:manual:`Change Streams </changeStreams/>`.
+Open a change stream by calling the ``watch()`` method on a ``Collection``, ``Db``, or ``MongoClient`` object. The
+change stream emits :manual:`change event </reference/change-events/>` documents when they occur.
 
-The ``watch()`` method optionally takes an **aggregation pipeline**  which
-consists of an array of **stages** as the first parameter to filter and
-transform the change events output as follows:. 
+The ``watch()`` method optionally takes an :manual:`aggregation pipeline </reference/operator/aggregation-pipeline/>`
```

Comment:
Same comment about linking to material external to the node driver docs early on in the page, applies to other instances below. Usually, the links are featured at the end of the section or page in a section for further information (see line 26-27 for an example of this redirection).